bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java

```diff
@@ -89,6 +101,9 @@ protected void internalPoll() throws FreeboxException {
                 }
             }
             updateChannelQuantity(SYS_INFO, UPTIME, uptime, Units.SECOND);
+        } else {
+            logger.debug("{}: poll with reachable={}", thing.getUID(), reachable);
+            updateStatus(ThingStatus.OFFLINE);
```

Comment:
Would it make sense to add a status description like "Player is not reachable" here?

Reply:
Good idea, done.